ui/src/app/certificate_requests/row.tsx

```diff
@@ -0,0 +1,32 @@
+const extractCSR = (csrPemString: string) => {
+    //TODO
+}
+
+const extractCert = (certPemString: string) => {
+    //TODO
+}
+
+type rowProps = {
+    id: number,
+    csr: string,
+    certificate: string
+}
+export default function Row({ id, csr, certificate }: rowProps) {
+    return (
+        <tr>
+            <td className="" data-test-column="id">{id}</td>
+            <td className="u-overflow--visible" data-test-column="details">
+                <p>CN: example.com</p>
+                <p>SAN: example.com, 127.0.0.1, 1.2.3.4.5.56</p>
+            </td>
+            <td className="" data-test-column="status">{certificate == "" ? "outstanding" : (certificate == "rejected" ? "rejected" : "certificate expiry date here")}</td>
```

Comment:
It's not quite clear why the expiry date is in the status of the CSR.

Reply:
I think it would be a nice touch to have some useful information here, since if there's a certificate attached that already implies that the CSR was approved.

There's also a [bunch](https://vanillaframework.io/docs/base/tables) of things we can do here, like have an expandable menu for details. 

I'd like to go over this in a call so we can come to a decision about what would be best here, since how this table looks is extremely subjective.

Reply:
After a meeting and a demo, decided to split this into 2 columns